src/main/kotlin/com/example/kbocombo/auth/infra/OAuthClients.kt

```diff
@@ -0,0 +1,26 @@
+package com.example.kbocombo.auth.infra
+
+import com.example.kbocombo.auth.domain.MemberInfo
+import com.example.kbocombo.member.domain.vo.SocialProvider
+
+class OAuthClients(
+    oAuthClients: Set<OAuthClient>
+) {
+    private val clients: Map<SocialProvider, OAuthClient> = oAuthClients.associateBy { it.getSocialProvider() }
```

Comment:
```suggestion


@Component
class OAuthClients(
    private val oAuthClients: Set<OAuthClient>
) {
    private val clients: Map<SocialProvider, OAuthClient> = oAuthClients.associateBy { it.getSocialProvider() }
    
@Configuration
class AuthConfig {

//    @Bean
//    fun oauthClients(clients: Set<OAuthClient>): OAuthClients {
//        return OAuthClients(oAuthClients = clients)
//    }
}    
```

AuthConfig 없이 그냥 OauthClients를 빈으로 선언하는거 어떤가요 간결하게


<img width="1057" alt="스크린샷 2025-01-09 오후 11 16 17" src="https://github.com/user-attachments/assets/7749f4af-990b-4630-8035-b194b9ee515e" />
빈 주입 완료 !

Reply:
👍👍👍👍